lib/helpers.coffee

```diff
@@ -32,9 +32,8 @@ module.exports = Helpers =
           resolve(data.stderr.join(''))
       if isNodeExecutable
         options.env ?= {}
-        for prop in process.env
-          if process.env.hasOwnProperty(prop) and prop isnt 'OS'
-            options.env[prop] = process.env[prop]
+        for prop,value of process.env
```

Comment:
and to @AsaAyers I know it's quite confusing, but it wasn't working for me and some other people that way, and the way it was enumerating it always resulted the env to be `{}` on linux and a valid object on windows. I know it's super weird, but this style change makes it work everywhere. I have confirmed it on a few machines, both before and after.